package.json

```diff
@@ -40,22 +42,23 @@
     "rimraf": "2.x",
     "sinon": "^1.10.0",
     "sinon-chai": "^2.5.0",
+    "source-map-support": "^0.4.0",
     "sqlite3": "^3.0.5",
     "tap-spec": "^4.0.0",
     "tape": "^4.0.0",
     "through": "^2.3.4",
     "webpack": "^1.13.0"
   },
   "scripts": {
-    "dev": "babel -L -D -w src/ --out-dir lib/",
-    "babel": "babel -L -D src/ --out-dir lib/",
+    "babel": "babel -L -D src/ --out-dir lib/ --source-maps=inline",
     "build": "./scripts/build.sh",
-    "tape": "node test/tape/index.js | tap-spec",
-    "test": "npm run babel && npm run jshint && istanbul --config=test/.istanbul.yml cover node_modules/mocha/bin/_mocha -- --check-leaks -t 5000 -b -R spec test/index.js && npm run tape",
-    "plaintest": "mocha --check-leaks -t 10000 -b -R spec test/index.js && npm run tape",
     "coveralls": "cat ./test/coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "jshint": "jshint test/. src/.",
-    "prepublish": "npm run build"
+    "dev": "babel -L -D -w src/ --out-dir lib/",
+    "lint": "eslint src/**",
+    "plaintest": "mocha --check-leaks -t 10000 -b -R spec test/index.js && npm run tape",
+    "prepublish": "npm run build",
+    "tape": "node test/tape/index.js | tap-spec",
+    "test": "npm run babel && istanbul --config=test/.istanbul.yml cover node_modules/mocha/bin/_mocha -- --check-leaks -t 5000 -b -R spec test/index.js && npm run tape"
```

Comment:
I have created another .eslintrc file for my test folder to allow mocha and other global helpers for example.